test/Test/Main.purs

```diff
@@ -105,15 +120,15 @@ testIntDivMod = do
       assert (msg <> "Remainder should be between 0 and `abs b`, got: " <> show r) $
         0 <= r && r < abs b
 
-testIntDegree :: AlmostEff
+testIntDegree :: Effect Unit
 testIntDegree = do
-    let bot = bottom :: Int
+    -- let bot = bottom :: Int
     assert "degree returns absolute integers" $ degree (-4) == 4
     assert "degree returns absolute integers" $ degree 4 == 4
-    assert "degree returns absolute integers" $ degree bot >= 0
-    assert "degree does not return out-of-bounds integers" $ degree bot <= top
+    -- assert "degree returns absolute integers" $ degree bot >= 0
```

Comment:
These as well

Reply:
Makes sense. Let's leave a comment and a TODO about this so the future implementors will find a thread to follow